Addition of a question (#6057)
MicrosoftDocs/windows-itpro-docs: windows/security/identity-protection/hello-for-business/hello-faq.md

## Can I use an external camera when my laptop is closed or docked?
No. Windows 10 currently only supports one Windows Hello for Business camera and does not fluidly switch to an external camera when the computer is docked with the lid closed.  The product group is aware of this and is investigating this topic further.

> ## Why is authentication failed after setting up PIN registration?

Reviewer: ```suggestion
## Why does authentication fail after setting up PIN registration?
```

> ## Why does authentication fail after setting up PIN registration?

Reviewer 2: This title is too generic since this is a specific failure case.

```suggestion
## Why does authentication fail immediately after provisioning Hybrid Key Trust?
```

> In hybrid mode a lot sync happened after enrollment for authentication to work. Authentication will only work when the sync between AD Connect server to Azure AD will be completed.This sync depends on the sync time setup for AD Connect server to Azure AD.

Reviewer: ```suggestion
In hybrid mode, a lot of sync happen after enrollment for authentication to work. Authentication will only work when the sync between the AD Connect server to Azure AD is completed. This sync depends on the sync time setup for AD Connect server to Azure AD.
```

> In hybrid mode, a lot of sync happen after enrollment for authentication to work. Authentication will only work when the sync between the AD Connect server to Azure AD is completed. This sync depends on the sync time setup for AD Connect server to Azure AD.

Reviewer: ```suggestion
In hybrid mode, a lot of sync happens after enrollment for authentication to work. Authentication will only work when the sync between the AD Connect server to Azure AD is completed. This sync depends on the sync time setup for AD Connect server to Azure AD.
```

> In hybrid mode, a lot of sync happens after enrollment for authentication to work. Authentication will only work when the sync between the AD Connect server to Azure AD is completed. This sync depends on the sync time setup for AD Connect server to Azure AD.

Reviewer 2: ```suggestion
In a hybrid deployment, a user's public key must sync from Azure AD to AD before it can be used to authenticate against a domain controller. This sync is handled by Azure AD Connect and will occur during a normal sync cycle.
```
